homeassistant/components/verisure/alarm_control_panel.py

```diff
@@ -45,6 +45,12 @@ def __init__(self):
     @property
     def name(self):
         """Return the name of the device."""
+        giid = hub.config.get(CONF_GIID)
+        if giid is not None:
+            return '{} alarm'.format(
+                    [i['alias'] for i in hub.session.installations if i['giid'] == giid][0]
```

Comment:
Would be good if you did a check first to see if the value is in the list of installations, and if it isn't wrote that to the log

Reply:
I'm thinking something along the lines:

`if giid is not None:
    d = {i['giid']: i['alias'] for i in hub.session.installations}
    if giid in d.keys():
        return '{} alarm'.format(d[giid])
    else: 
         DO SOME LOGGING`

That will also solve line length issues.

Sounds good? In that case I'll submit update pull request

Reply:
Looks good!